docs/run_flask.md

```diff
@@ -0,0 +1,39 @@
+# How to run this flask app
+
+## Create python environment
+
+
+```sh
+# research_lab/
+
+python -m venv env
```

Comment:
change this please, we are using `conda` as env manager 

https://docs.conda.io/projects/conda/en/latest/_downloads/843d9e0198f2a193a3484886fa28163c/conda-cheatsheet.pdf

Reply:
this .md file should be deleted